### PR TITLE
add Jekyll site (#7) 

### PR DESCRIPTION
**Pull request by housseindjirdeh**
http://hnpwa.surge.sh/

Adds the site in a separate directory (`site`) (for Issue #7)

- If you would like to add Google Analytics tracking to the site, just add a `google_analytics` attribute to `config.yml` with your tracking code and build the site under production

Again, let me know if there's anything you would like me to change :)

### CODE REVIEW

File: site/_apps/angular.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+weight: 5
+title:  Angular
+logo: /assets/images/angular-logo.svg
+lighthouse: 100/100
+interactive-em: 5.8s
+interactive-faster-3g: 3.48s
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170423_BN_32H&run=1
+wpt-em-link: https://www.webpagetest.org/result/170420_46_67R/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170328_N6_761d4f12fce28adc5163a3d056ce0af6/
+image: /assets/images/angular2hn-mobile.png
```
Comment (addyosmani) [inline site/_apps/angular.txt]: I wonder if we can trim these down a little:

image

A quick pass-through ImageOptim seems like it'd save us about 35% on some of these:

image

I've also seen a ~55% win using https://compressor.io/compress. We can do those optimizations as part of this PR or work on it in a separate issue.

File: site/_apps/preact.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+weight: 2
+title:  Preact
+logo: /assets/images/preact-logo.svg
```
Comment (addyosmani) [inline site/_apps/preact.txt]: Thanks for using the SVGs!

File: site/_apps/preact.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+lighthouse: 99/100
+interactive-em: 2.4s
+interactive-faster-3g: 1.8s
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170423_RE_31X&run=1
+wpt-em-link: https://www.webpagetest.org/result/170420_73_54G/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170328_SX_2b62ae440379075b887f472c02a3e0a3/
+image: /assets/images/preacthn-mobile.png
+app-link: https://hn.kristoferbaxter.com/
```
Comment (addyosmani) [inline site/_apps/preact.txt]: @kristoferbaxter as an author, do you feel it is sufficient to link to the implementation or should we also aim to link to the GitHub repo from these listings?

File: site/_contributors/addy.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+weight: 1
+name: Addy
+image: /assets/images/addy.png
```
Comment (addyosmani) [inline site/_contributors/addy.txt]: These look great. The only comment I have would echo the earlier one about image compression :)

File: site/_apps/angular.txt (hunk 1 above)
Comment (housseindjirdeh) [inline site/_apps/angular.txt]: You're right, they're all quite large! used `compressor` to trim them down (for some of them ~70% reduction in size :O)

screen shot 2017-04-23 at 11 05 04 pm

File: site/_apps/preact.txt (hunk 3 above)
Comment (kristoferbaxter) [inline site/_apps/preact.txt]: The results are quite important, but for the intended audience code speaks volumes.

I agree with a link to source on the page.

File: site/_contributors/addy.txt (hunk 4 above)
Comment (housseindjirdeh) [inline site/_contributors/addy.txt]: Compresser.io wasn't giving much wins for these images, so just searched around and replaced them with much smaller images.

File: site/_apps/angular.txt (hunk 1 above)
Comment (addyosmani) [inline site/_apps/angular.txt]: \o/

File: site/_apps/preact.txt (hunk 3 above)
Comment (housseindjirdeh) [inline site/_apps/preact.txt]: Thanks @kristoferbaxter, and you're right it only makes sense to add a link to their sources.

Added another button to view their GitHub repos.

screen shot 2017-04-24 at 6 27 20 pm
screen shot 2017-04-24 at 6 27 44 pm

Comment (kristoferbaxter) [inline site/_apps/preact.txt]: Awesome! Looks great.

**Comment by sindresorhus** [comment]
Instead of including the images of the authors, how about we instead use their GitHub avatar image?

File: site/_apps/preact.txt (hunk 3 above)
Comment (addyosmani) [inline site/_apps/preact.txt]: Looks awesome. Thanks, @housseindjirdeh!

File: site/_contributors/addy.txt (hunk 4 above)
Comment (addyosmani) [inline site/_contributors/addy.txt]: Thanks. Wdyt of @sindresorhus's comment below about using GH avatars directly?

**Comment by housseindjirdeh** [comment]
@sindresorhus yeah that probably makes more sense actually, fixed it :)

**Comment by chrisdwheatley** [comment]
Can't believe how quickly this has come together, nice work @housseindjirdeh!

I really want to share this on [PWA Weekly](https://pwaweekly.com) but guessing it's not quite ready yet. Can I help contribute by adding a manifest & service worker or is that all in hand?

**Comment by housseindjirdeh** [comment]
@chrisdwheatley thank you 😄 And yes that's a great idea! I didn't get a chance to add a SW or manifest file in this PR so you can definitely help there. 

@addyosmani @sindresorhus what do you think of having that included in the site as a separate PR?

**Comment by addyosmani** [comment]
@housseindjirdeh That works for me 👍  Created https://github.com/tastejs/hacker-news-pwas/issues/13 to track switching to GH avatars.

@@chrisdwheatley I might suggest we can work on any PWA related pieces in follow up PRs. It would be great to create issues for them so we can chat about strategy (shouldn't take long at all)

**Comment by housseindjirdeh** [comment]
@addyosmani oops sorry for not being more clear, already pushed an update to use GH avatars instead of locally saved ones. I was referring to having a SW/manifest file in a separate PR as you just mentioned :)

**Comment by addyosmani** [comment]
No worries! I should have double checked the most recent commits :) As a first pass this LGTM to land and iterate on.

**Comment by sindresorhus** [comment]
Just wanted to say that the website turned out great. Superb work @housseindjirdeh :)

**Comment by housseindjirdeh** [comment]
Thank you @sindresorhus, I'm glad you liked how it turned out :)

**Pull request merged**